bigquery/unit_tests/test_table.py

```diff
@@ -1129,9 +1133,12 @@ def test_fetch_data_w_alternate_client(self):
         table = self._makeOne(self.TABLE_NAME, dataset=dataset,
                               schema=[full_name, age, voter, score])
 
-        rows, total_rows, page_token = table.fetch_data(client=client2,
-                                                        max_results=MAX,
-                                                        page_token=TOKEN)
+        iterator = table.fetch_data(
+            client=client2, max_results=MAX, page_token=TOKEN)
```

Comment:
Preferred over what? (Reading PEP8 should clear up most of this question, not sure how familiar you are with that document.)

Essentially there are two choices here, both of which are perfectly valid:

```python
        iterator = table.fetch_data(
            client=client2, max_results=MAX, page_token=TOKEN)
```

OR

```python
        iterator = table.fetch_data(client=client2, max_results=MAX,
                                    page_token=TOKEN)
```